src/components/receipt-view/data-section/DataSection.js

```diff
@@ -61,12 +61,12 @@ const DataSection = ({ cartState }) => {
       item
       container
       alignItems="center"
-      sm={ratio >= minRatio.body ? 12 : 10}
+      xs={ratio >= minRatio.body ? 12 : 10}
       spacing={1}
       className={styles.dataSectionBox}>
       <Grid
         item
-        sm={(width - min) / width >= minRatio.body ? 12 : 10}
+        xs={(width - min) / width >= minRatio.body ? 12 : 10}
```

Comment:
Wait, I don't understand what this is doing. Will you explain it to me?
Couldn't the same thing be achieved by doing: 
xs={10}
sm={12}

Reply:
Yeah it totally could. I think when I did this I didn't understand what the `xs` vs `sm` was really doing. That's a quick enough change, I'll go ahead and do that now before we merge it

Reply:
So this is actually more precise than using the breakpoints, I think we should keep it for now? I spent a bit of time on it and couldn't get it to work as well as it does now. Might not be worth fixing for Version 1

Reply:
Sounds good